### PR TITLE
Install instruction for bullseye

### PR DESCRIPTION
**Pull request by chayim**
closes #70

### CODE REVIEW

File: docs/stack/get-started/install/linux.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ weight: 1
 
 
 ### From the official Debian/Ubuntu APT Repository
-You can install recent stable versions of Redis Stack from the official packages.redis.io APT repository. The repository currently supports Ubuntu Xenial (16.04), Ubuntu Bionic (18.04), and Ubuntu Focal (20.04) on x86 processors. Add the repository to the apt index, update it and install:
+You can install recent stable versions of Redis Stack from the official packages.redis.io APT repository. The repository currently supports Debian Bullseye (11), Ubuntu Xenial (16.04), Ubuntu Bionic (18.04), and Ubuntu Focal (20.04) on x86 processors. Add the repository to the apt index, update it and install:
```
Comment (nermiller) [inline docs/stack/get-started/install/linux.md:11]:
```suggestion
You can install recent stable versions of Redis Stack from the official packages.redis.io APT repository. The repository currently supports Debian Bullseye (11), Ubuntu Xenial (16.04), Ubuntu Bionic (18.04), and Ubuntu Focal (20.04) on x86 processors. Add the repository to the apt index, update it, and install it:
```

**Review by nermiller — APPROVED** [review]
LGTM! just a minor nitpick, @chayim

**Comment by chayim** [comment]
@nermiller I thought I sent this for re-review after including your comments. Thoughts?

**Comment by nermiller** [comment]
LGTM!

**Review by nermiller — APPROVED** [review]
LGTM!

**Pull request merged**